addons/airway/CfgWeapons.hpp

```diff
@@ -38,4 +38,15 @@ class CfgWeapons {
             mass = 7;
         };
     };
+    class kat_suction: ACE_ItemCore {
+        scope = 2;
+        author = "Apricot_ale";
+        displayName= CSTRING(Suction_Display);
+        descriptionShort = CSTRING(Suction_Desc_Short);
+        picture = QPATHTOF(ui\suction_normal.paa);
+        ACE_isMedicalItem = 1;
+        class ItemInfo: CBA_MiscItem_ItemInfo {
+            mass = 1.5;
```

Comment:
change to a lower more realistic number. 1.5 is quite big.

Reply:
changed